ludwig/utils/audio_utils.py

```diff
@@ -64,8 +63,11 @@ def read_audio_from_path(path: str) -> Optional[TorchAudioTuple]:
 
     Useful for reading from a small number of paths. For more intensive reads, use backend.read_binary_files instead.
     """
-    bytes_obj = get_bytes_obj_from_path(path)
-    return read_audio_from_bytes_obj(bytes_obj)
+    try:
+        return torchaudio.backend.sox_io_backend.load(path)
+    except Exception as e:
```

Comment:
Is there a more specific exception we could catch here?

Reply:
I’m noticing that the same kind of Exception throwing is occurring in the block on line 76. I’m not sure what specific exceptions are going to be caught here, but it appears that we want to be general about it based on the other test? Do you think it would be better to try to catch a more specific one?

Reply:
I'm typically a fan of catching specific exception types so that we don't suppress errors we need to fix. For now, this shouldn't block the merge.